README.md

```diff
@@ -120,19 +120,27 @@ _As far as I know, there's nothing we can do about those two limitations. Please
 But you generally only use stubs during the development phase and want to remove your stubs when submitting to the AppStore. So be careful to only include `OHHTTPStubs` in your test targets, or only use it in `#if DEBUG` portions, to avoid forgetting to remove it when the time comes that you release for the AppStore!
 
 
-## About OHHTTPStubs own's Unit Tests
+# Building OHHTTPStubs
 
-If you want to be able to run `OHHTTPStubs`' Unit Tests, be sure you cloned the [`AFNetworking`](https://github.com/AFNetworking/AFNetworking/) submodule (by using the `--recursive` option when cloning your repo, or using `git submodule init` and `git submodule update`) as it is used by some of `OHHTTPStubs` unit tests.
+## Building OHHTTPStubs
+To build (and run unit tests) `OHHTTPStubs`, use cocoapods. We are using [`AFNetworking`](https://github.com/AFNetworking/AFNetworking/) for unit tests only. No other dependencies are used for `OHHTTPStubs`.
 
-> Note: this need for submodule for the test target is bound to change very soon with [#86](https://github.com/AliSoftware/OHHTTPStubs/issues/86)
+```bash
+pod install
+open OHHTTPStubs.xcworkspace
+```
 
-This submodule is only useful for `OHHTTPStubs`' own Unit Tests, that are testing its usage with `AFNetworking`: **you don't need the submodule to use `OHHTTPStubs`** and `OHHTTPStubs` has no dependency on `AFNetworking` itself.
+## Building OHHTTPStubsDemo
+To build (and run unit tests) `OHHTTPStubsDemo` use cocoapods.
 
+```bash
+cd OHHTTPStubsDemo
+pod install
+open OHHTTPStubsDemo.xcworkspace
```

Comment:
BTW using the Development Pods feature will allow us to skip this `pod install` step (as the Dev Pod will directly be linked internally without the need to run `pod install` to fetch it).

That's the beauty of Development Pods. And that will allow the `pod try` command to continue to work transparently even.